assets/components/src/button/style.scss

```diff
@@ -7,16 +7,17 @@
 .muriel-button {
 	text-align: center;
 	font-size: 14px;
-	line-height: 18px;
+	line-height: 21px;
 	font-weight: 500;
 	color: $muriel-gray-700;
 
 	&.is-button {
 		background-color: $muriel-white;
 		border-color: #cccccc;
-		border-radius: 3px;
+		border-radius: 4px;
 		height: auto;
-		padding: 16px 32px;
+		line-height: 21px;
+		padding: 7px 14px 9px;
```

Comment:
The change in `Button` padding is almost certainly the most noticeable aspect of this PR. Originally the buttons were designed to Muriel spec, including the fairly significant padding. I'm all for trimming them down, but would like to know where the new padding comes from? Is it purely an aesthetic improvement or does it draw from any particular design framework?

Reply:
I copied the padding from Calypso... Thinking about it now, it doesn't make much sense since our vertical rhythm is currently based on a 21px grid 🤦‍♂ 

If we want to keep the large buttons, I think we can have 2 types of buttons: "default" (42px) and "large" (63 or even 84px).

Reply:
See: 2c2d78d